Makefile.dune

```diff
@@ -44,6 +48,22 @@ ocheck: voboot
 clean:
 	dune clean
 
+# Dune bootstrap targets:
+
+DUNE_BOOT_VER=1.3.0
+DUNE_TBZ=dune-$(DUNE_BOOT_VER).tbz
+DUNE_BOOT_PATH=_dune/dune-$(DUNE_BOOT_VER)/_build/install/default/bin
+bootstrap:
+	mkdir -p _dune
```

Comment:
On OSX `curl` used to come by default, while `wget` did not. Not sure which is the best way to use the right one in this one liner.

Reply:
Yeah I was doubting about that, I dunno, will take either of them.